Lib/platform.py

```diff
@@ -972,6 +1003,10 @@ def uname():
         system = 'Windows'
         release = 'Vista'
 
+    # Normalize responses on iOS
+    if sys.platform == 'ios':
+        system, release, machine, _ = ios_ver()
```

Comment:
I missed the platform.machine discrepancy between the historical implementation we've used for BeeWare, and what the PEP converged on; I've corrected this.

I've also added documentation for the `uname` discrepancy (on both the `os` and `platform` side), and iOS tests for the `uname` return values that are iOS specific.

I agree there are no meaningful differences between iOS and iPadOS; however, there is a *branding* distinction. I'd argue the fact that `sys.platform == "ios"` for both is *more* reason to return a user-facing value for `platform.system`; if you care about "is this a Apple i-device", you can use `sys.platform`; if you care about what the device OS calls itself, you can use `platform.system()`. However, if the core team reviewers feel strongly otherwise, I won't push back too hard - as you say, the value is available in `ios_ver()`.

Reply:
Since we're following the principle of making the `platform` APIs return user-facing names, I guess that makes sense.